engine/src/main/resources/assets/ui/menu/joinGameScreen.ui

```diff
@@ -1,338 +1,374 @@
+            ]
+          },
+          {
+            "type": "RelativeLayout",
+            "family": "description",
+            "contents": [
+              {
+                "type": "ColumnLayout",
+                "id": "simpleItems",
+                "columns": 2,
+                "column-widths": [
+                  0.25,
+                  0.75
+                ],
+                "extendLast": true,
+                "verticalSpacing": 8,
+                "contents": [
+                  {
+                    "type": "UILabel",
+                    "text": "${engine:menu#server-name}: "
+                  },
+                  {
+                    "type": "UILabel",
+                    "id": "name",
+                    "text": "Server Name Goes Here"
+                  },
+                  {
+                    "type": "UILabel",
+                    "text": "${engine:menu#server-owner}: "
+                  },
+                  {
+                    "type": "UILabel",
+                    "id": "owner",
+                    "text": "Server Owner Goes Here"
+                  },
+                  {
+                    "type": "UILabel",
+                    "text": "${engine:menu#server-address}: "
+                  },
+                  {
+                    "type": "UILabel",
+                    "id": "address",
+                    "text": "Server Address Goes Here"
+                  },
+                  {
+                    "type": "UILabel",
+                    "text": "${engine:menu#server-port}: "
+                  },
+                  {
+                    "type": "UILabel",
+                    "id": "port",
+                    "text": "Server Port Goes Here"
+                  },
+                  {
+                    "type": "UILabel",
+                    "text": "${engine:menu#game-worlds}: "
+                  },
+                  {
+                    "type": "UILabel",
+                    "id": "worlds",
+                    "text": "World Info Goes Here"
+                  },
+                  {
+                    "type": "UILabel",
+                    "text": "${engine:menu#module-list}: "
+                  },
+                  {
+                    "type": "ScrollableArea",
+                    "content": {
+                      "type": "UILabel",
+                      "id": "modules",
+                      "text": "Module Info Goes Here"
+                    }
+                  }
+                ],
+                "layoutInfo": {
+                  "position-horizontal-center": {},
+                  "position-top": {
+                    "target": "TOP",
+                    "offset": 8
+                  },
+                  "position-bottom": {
+                    "target": "TOP",
+                    "widget": "buttonRow",
+                    "offset": 8
+                  }
+                }
+              },
+              {
+                "type": "UIButton",
```

Comment:
Please delete this button, as it's a duplicate squeezed in between the other two.

![image](https://cloud.githubusercontent.com/assets/13783592/19149299/cebe6a84-8bc9-11e6-9a75-1a6b6031fb04.png)